modules/core/arrow-test/src/main/kotlin/arrow/test/laws/TraverseLaws.kt

```diff
@@ -47,17 +48,19 @@ object TraverseLaws {
     )
 
   fun <F> Traverse<F>.identityTraverse(FF: Functor<F>, cf: (Int) -> Kind<F, Int>, EQ: Eq<Kind<F, Int>>) = Id.applicative().run {
+    val idApp = this
     forAll(genFunctionAToB<Int, Kind<ForId, Int>>(genConstructor(genIntSmall(), ::Id)), genConstructor(genIntSmall(), cf)) { f: (Int) -> Kind<ForId, Int>, fa: Kind<F, Int> ->
-      fa.traverse(this, f).value().equalUnderTheLaw(FF.run { fa.map(f).map { it.value() } }, EQ)
+      fa.traverse(idApp, f).extract().equalUnderTheLaw(FF.run { fa.map(f).map { it.extract() } }, EQ)
```

Comment:
I'm not completely convinced about extract instead of value, but I also remember having a discussion about this before. Whether we wanted to deprecate value for id or not.